src/components/Comments.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from "../../api";
+
+function Comments({singleArticle, formattedData}) {
+  const [comments, setComments] = useState([]);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getComments(article_id).then((data) => {
+      setComments(data);
+    });
+  }, [article_id]);
```

Comment:
This is great, it would also be really useful to have `loading` state here so a user knows the comments are still being fetched